code/web/interface/themes/responsive/MyAccount/list.tpl

```diff
@@ -125,6 +125,7 @@
 									{/if}
 									<button value="printList" id="FavPrint" class="btn btn-sm btn-default listViewButton" onclick='return AspenDiscovery.Lists.printListAction()'>{translate text='Print List' isPublicFacing=true}</button>
 									<a id="FavExport" class="btn btn-sm btn-default listViewButton" href="/MyAccount/AJAX?method=exportUserList&listId={$userList->id}">{translate text='Export List to CSV' isPublicFacing=true}</a>
+									<a id="FavEportRis" class="btn btn-sm btn-default listViewButton" href="/MyAccount/AJAX?method=exportUserListRIS&listId={$userList->id}">{translate text='Export List to RIS' isPublicFacing=true}</a>
```

Comment:
Tiny picky comment.. `FavEportRis` => `FavExportRis` might be slightly clearer.. looks like a typo/spelling right now ;P